public/app/features/auth-config/AuthProvidersListPage.tsx

```diff
@@ -52,8 +52,9 @@ export const AuthConfigPageUnconnected = ({
     reportInteraction('authentication_ui_provider_clicked', { provider: providerType, enabled });
   };
 
-  // filter out saml from sso providers because it is already included in availableProviders
+  // filter out saml and ldap from sso providers because it is already included in availableProviders
   providers = providers.filter((p) => p.provider !== 'saml');
+  providers = providers.filter((p) => p.provider !== 'ldap');
```

Comment:
This already existed at line 60. Lets keep only one. Maybe just update the comment from the second one.

Reply:
changed as requested